extension/src/sidebar-outer.css

```diff
@@ -1,7 +1,20 @@
-#promnesia-sidebar.promnesia {
+/* Resets for sites */
+body.promnesia-sidebar-active {
+    transform: none;
+}
+.promnesia-sidebar-active #promnesia-frame {
+    margin: 0;
+    padding: 0;
+    max-width: none;
```

Comment:
hmm what's it for?

Reply:
for issue https://github.com/karlicoss/promnesia/issues/102